hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/compactions/CurrentHourProvider.java

```diff
@@ -37,7 +38,8 @@
     }
   }
 
-  private static Tick nextTick() {
+  @VisibleForTesting
+  protected static Tick nextTick() {
```

Comment:
Just remove the protected qualifier? We just want it to be package private right?

Reply:
Will fix later, thanks.